source/codegen/metadata/nitclk/config.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-TClk API metadata version 255.0.0d0
+config = {
+    'api_version': '255.0.0d0',
+    'c_header': 'niTClk.h',
+    'c_function_prefix': 'niTClk_',
+    'service_class_prefix': 'NiTClk',
+    'java_package': 'com.ni.grpc.tclk',
+    'csharp_namespace': 'NationalInstruments.Grpc.TClck',
```

Comment:
The `TClck` is inconsistent with `tclk` and `nitclk`. Did you mean to make this `TClk`? Would `TClock` be better?

Reply:
Yep, that was typo. I have updated it to `TClk`. Expanding to `TClock` does seem better and matches with typical C# conventions, but the APIs and attributes we are exposing here still follow C convention and have `TClk` in their names. Having mix of both would be inconsistent.